src/app.scss

```diff
@@ -6,3 +6,4 @@
 @import './components/TutorialHeader/tutorial-header.scss';
 @import './content/LandingPage/landing-page.scss';
 @import './content/RepoPage/repo-page.scss';
+@import './components//Info/info.scss';
```

Comment:
stray double-slash will be ignored by this preprocessor but sometimes double-slash is interpreted as a forced-root directive